src/Castle.Core.Tests/project.json

```diff
@@ -19,13 +19,18 @@
   },
   "compilationOptions": {
     "warningsAsErrors": true,
-    "define": [ "FEATURE_XUNITNET" ]
+    "define": [ "FEATURE_XUNITNET" ],
+    "keyFile": "../../buildscripts/CastleKey.snk"
   },
+  "resource": [
+    "Core.Tests/Ressources/MoreRes/TestRes.resx"
```

Comment:
Yes the test is using ResourceManager. It should be supported:(https://github.com/dotnet/corefx/tree/c02d33b18398199f6acc17d375dab154e9a1df66/src/System.Resources.ResourceManager/tests)  We will need to prevent the auto-generation of the Designer.cs since the generated code use `GetType` while we need to use `GetTypeInfo()`.